src/app/molecules/message/Message.jsx

```diff
@@ -802,7 +803,7 @@ function Message({
         )}
         {isEdit && (
           <MessageEdit
-            body={body}
+            body={(customHTML ? html : plain)(customHTML || body, { kind: 'edit' }).plain}
```

Comment:
This condition feels confusing. we can just simply do `customHTML ? html(....) : plain( .... )`